Lib/KeychainAccess/Keychain.swift

```diff
@@ -632,9 +638,11 @@ public class Keychain {
                     if let account = credentials[kSecAttrAccount as! String] as? String {
                         credential["account"] = account
                     }
+                    #if os(iOS)
                     if let password = credentials[kSecSharedPassword.takeUnretainedValue() as! String] as? String {
```

Comment:
`kSecSharedPassword` is undefined on OS X.
Simply exclude it by using `#if os(iOS)`